sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2_AnalyzeSentimentBatch.cs

```diff
@@ -56,6 +56,7 @@ public void AnalyzeSentimentBatch()
                 if (result.ErrorMessage != default)
                 {
                     Debug.WriteLine($"On document (Id={document.Id}, Language=\"{document.Language}\", Text=\"{document.Text}\"):");
```

Comment:
I think I changed some of these somewhere, too ... maybe in the UX study branch?  Thanks for catching this!  Let's move the "On document..." lines outside the if (error) block, so it's printed for both success and error cases.